webapp/src/containers/createIncident/styles.scss

```
@@ -0,0 +1,56 @@
+.incident-body {
+    padding: 25px 32px 0px 32px;
```

```suggestion
    padding: 25px 32px 0 32px;
```
no need to put any unit for `0` in CSS